src/GeoTagFunctions.java

```diff
@@ -362,38 +351,10 @@ public static boolean removeGeoTagData(File jpeg) {
         File resultFile = new File("./assets/results/editted-" + jpeg.getName());
 
     	try {
-    		final int LATITUDE_REFERENCE_TAG = 1;
-    	    final int LATITUDE_TAG = 2;
-    	    final int LONGTITUDE_REFERENCE_TAG = 3;
-    	    final int LONGTITUDE_TAG = 4;
-    		
-    	    // copy the original information
-    	    GeoTagFunctions.readImageMeta(jpeg);
-    		
-    	    TiffOutputSet outputSet = null;
-    		
-    		if (exif != null) {
-                // get a copy of exif data to be muted.
-                outputSet = exif.getOutputSet();
-            } 
-            else {
-            	outputSet = new TiffOutputSet();
-            	
-            	// avoid empty exif data that cause null pointer error in output
-            	outputSet.getOrCreateExifDirectory();
-            }
-    		
-    		// remove latitude and longitude value.
-    		Utility.displayProcessing("remove-geotag");
-
-    		outputSet.removeField(LATITUDE_REFERENCE_TAG);
-            outputSet.removeField(LATITUDE_TAG);
-            outputSet.removeField(LONGTITUDE_REFERENCE_TAG);
-            outputSet.removeField(LONGTITUDE_TAG);
+    		Jpeg image = new Jpeg(jpeg);
```

Comment:
This line here causes an exception. Unable to update or remove GeoTag.

Reply:
Could you take a screen shot?

Reply:
![Uploading Screen Shot 2021-12-01 at 5.51.14 PM.png…]()